i18n/fr/docusaurus-plugin-content-docs/version-20.10/releases/centreon-commercial-extensions.md

```diff
@@ -196,6 +196,15 @@ Référez-vous à la [documentation dédiée](../graph-views/install.md#centreon
 
 ## Centreon BAM
 
+### 20.10.6
+
+Release date: `July 19, 2022`
+
+#### Bugfixes
```

Comment:
```suggestion
#### Bug fixes
```

Reply:
that's true that is  mistake in English but we should fix it in all release notes to be consistent